src/minichain/network.py

```
@@ -0,0 +1,1075 @@
+    async def _peer_reader_loop(self, peer_id: str) -> None:
+        connection = self._connections.get(peer_id)
+        if connection is None:
+            return
+        try:
+            while self._running:
+                message = await self._read_message(connection.reader, eof_ok=True)
+                if message is None:
+                    break
+                await self._handle_peer_message(peer_id, message)
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)